Description: Fix build errors on Armhf during
"build instrumented compiler phase".
Author: Nathan Pratta Teodosio <nathan.teodosio@canonical.com>
Forwarded: not-yet
--- a/third_party/llvm/llvm/utils/gn/build/BUILD.gn
+++ b/third_party/llvm/llvm/utils/gn/build/BUILD.gn
@@ -110,7 +110,7 @@
       # no point in passing -gsplit-dwarf here.
     }
     if (is_optimized) {
-      cflags += [ "-O3" ]
+      cflags += [ "-O1" ]
     }
     cflags += [ "-fdiagnostics-color" ]
     if (use_lld) {
